test/ui/ResultLayoutTest.ts

```diff
@@ -29,6 +30,7 @@ export function ResultLayoutTest() {
             return builder;
           }
         });
+        $$(test.env.root).trigger(InitializationEvents.afterComponentsInitialization);
```

Comment:
Instead of trigerring this event repeatedly, could we not group the test that need this event together in a "describe" function, and only put it once ?